src/main/java/ca/uhn/fhir/jpa/starter/AppProperties.java

```diff
@@ -556,6 +557,18 @@ public boolean getInstall_transitive_ig_dependencies() {
 	public void setInstall_transitive_ig_dependencies(boolean install_transitive_ig_dependencies) {
 		this.install_transitive_ig_dependencies = install_transitive_ig_dependencies;
 	}
+	
+	public boolean getReload_existing_implementationguides() {
+		return reload_existing_implementationguides;
+	}
+	
+	public void setReload_existing_implementationguides(boolean reload_existing_implementationguides) {
+		this.reload_existing_implementationguides = reload_existing_implementationguides;
+	}
+
+	public List<String> getLocal_base_urls() {
```

Comment:
This was merged at the wrong place. This same function was declared later in the upstream repo.